lld/COFF/Config.h

```diff
@@ -17,6 +17,7 @@
 #include "llvm/Object/COFF.h"
 #include "llvm/Support/CachePruning.h"
 #include "llvm/Support/VirtualFileSystem.h"
+#include "llvm/LTO/LTO.h"
```

Comment:
Please don't add new includes of LTO.h to other headers, it is very expensive in compile time:
https://github.com/llvm/llvm-project/commit/f24c3352c97b864ae6c9851949497b8f880133f6

I would recommend that you forward declare the enum to avoid the need for this, but I see that LTOKind is a class member, which makes that harder.

Reply:
Maybe that's why ELF's implementation redeclared those enums...